src/LibHdb++Cassandra.cpp

```diff
@@ -2300,71 +1077,94 @@ void HdbPPCassandra::throw_execute_exception(string message, string query, CassE
 void HdbPPCassandra::set_cassandra_logging_level(string level)
 {
     if (level == "DISABLED")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_DISABLED);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_DISABLED);
 
     else if (level == "CRITICAL")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_CRITICAL);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_CRITICAL);
 
     else if (level == "ERROR")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_ERROR);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_ERROR);
 
     else if (level == "WARN")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_WARN);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_WARN);
 
     else if (level == "INFO")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_INFO);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_INFO);
 
     else if (level == "DEBUG")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_DEBUG);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_DEBUG);
 
     else if (level == "TRACE")
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_TRACE);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_TRACE);
 
     else
     {
-        cass_log_set_level(cassandra_logging_level = CASS_LOG_DISABLED);
+        cass_log_set_level(_cassandra_logging_level = CASS_LOG_DISABLED);
         LOG(Error) << "ERROR invalid cassandra driver logging level" << endl;
         LOG(Error) << "Log level set by default to: DISABLED" << endl;
     }
 
-    LOG(Debug) << "Cassandra driver logging set to: ";
+    LOG(Info) << "Cassandra driver logging set to: ";
 
-    if (cassandra_logging_level == CASS_LOG_DISABLED)
+    if (_cassandra_logging_level == CASS_LOG_DISABLED)
+    {
+        LOG(Info) << "DISABLED" << endl;
+    }
+    else
     {
-        LOG(Debug) << "DISABLED" << endl;
+        LOG(Info) << cass_log_level_string(_cassandra_logging_level) << endl;
     }
+}
+
+//=============================================================================
+//=============================================================================
+void HdbPPCassandra::set_library_logging_level(std::string level)
+{
+    if(level == "ERROR")
+        LoggerClass::Log::LogLevel() = Error;
+    else if(level == "WARNING")
+        LoggerClass::Log::LogLevel() = Warning;
+    else if(level == "INFO")
+        LoggerClass::Log::LogLevel() = Info;
+    else if(level == "DEBUG")
+        LoggerClass::Log::LogLevel() = Debug;
+    else if(level == "DISABLED")
+        LoggerClass::Log::LogLevel() = Disabled;
     else
     {
-        LOG(Debug) << cass_log_level_string(cassandra_logging_level) << endl;
+        LOG(Error) << "Invalid or no logging logging level selected, setting to default: ERROR" << endl;
```

Comment:
"no logging logging level" => "no logging level"

Reply:
Hey, thats the logging for the logging level ;)

Fixed.